weave/weave_client.py

```diff
@@ -561,6 +572,16 @@ def delete_call(self, call: Call) -> None:
             )
         )
 
+    @trace_sentry.global_trace_sentry.watch()
+    def rename_call(self, call: Call, display_name: str) -> None:
```

Comment:
Seeing this pattern makes me wonder, "should all operations follow this pattern?:"

* LOWEST LEVEL: `trace_server_interface` has capability directly or indirectly via other capabilities
* `remote_http_trace_server` -> exposes http interface in language specific binding (could be generated someday)
* `weave_client`: contains some `[action]_[class](self, obj: Class, params)` to do the action operating on objects
* HIGHEST LEVEL `[Class].action(params)` - OO style of action

Maybe so? I like that clarity in design.